docs/content/filtering.md

```diff
@@ -562,6 +562,8 @@ Other filter flags (`--include`, `--include-from`, `--exclude`,
 trailing whitespace is stripped from the input lines. Lines starting
 with `#` or `;` are ignored.
 
+`--files-from` followed by `-` reads files list from standard input.
```

Comment:
If I'm not mistaken, the following is more correct spelling:

```suggestion
`--files-from` followed by `-` reads file list from standard input.
```

Reply:
`reads the list of files from standard input.`

Would be my preferred wording.